tests/Find-DbaInstance.Tests.ps1

```diff
@@ -42,4 +42,4 @@ Describe "$CommandName Integration Tests" -Tag "IntegrationTests" {
             $results.SqlConnected | Should -Be $true
```

Comment:
> Expected $true, but got @($true, $true, $true, $false, $true).

This is the result of AppVeyor's environment, which has a minimum of 3 versions of SQL Server running on it and then localDB is there because of Visual Studio so our command finds them all. 

The test should be changed to filter the `$results` where it only checks the instance3 for the test and verifies the properties for that instance only.